docs/api/sigpig.md

```
@@ -387,6 +386,10 @@ websites가 포함된다면, 기존의 websites는 모두 삭제되고 새로운
 * **Status Codes**:
 
   * `204 No Content`
+   * `400 Bad Request`: sig 상태가 탈퇴 가능한 상태가 아닐 때
```

markdownlint-cli2 (0.22.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 3

(MD005, list-indent)